index.html

```diff
@@ -60,9 +60,162 @@ <h3>I can help you build a product , feature or website Look through some of my
+            <h1 class="s4h1"> Profesional Art Printing Data</h1> 
+            <p class="p2">  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+  
+                   
+
+          <ul class="lan2" >
+              <li><img src="image/html2.png" alt=""></li>
+              <li><img src="image/bootstrap2.png" alt=""></li>
+              <li><img src="image/ruby2.png" alt=""></li>
+
+          </ul>
+
+          <button class="btn2">See Project </button> </div>
+
+          <div class="project2" id="project22">
+            <h1 class="s4h1"> Profesional Art Printing Data</h1> 
+            <p class="p2">  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+  
+                   
+
+          <ul class="lan2" >
+              <li><img src="image/html2.png" alt=""></li>
+              <li><img src="image/bootstrap2.png" alt=""></li>
+              <li><img src="image/ruby2.png" alt=""></li>
+
+          </ul>
+
+          <button class="btn2">See Project </button> </div>
+
+          <div class="project2" id="project21">
+            <h1 class="s4h1"> Profesional Art Printing Data</h1> 
+            <p class="p2">  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+  
+                   
+
+          <ul class="lan2" >
+              <li><img src="image/html2.png" alt=""></li>
+              <li><img src="image/bootstrap2.png" alt=""></li>
+              <li><img src="image/ruby2.png" alt=""></li>
+
+          </ul>
+         <button class="btn2">See Project </button> </div>
+
+
+          <div class="project2" id="project20">
+            <h1 class="s4h1"> Profesional Art Printing Data</h1> 
+            <p class="p2">  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+  
+                   
+
+          <ul class="lan2" >
+              <li><img src="image/html2.png" alt=""></li>
+              <li><img src="image/bootstrap2.png" alt=""></li>
+              <li><img src="image/ruby2.png" alt=""></li>
+
+          </ul>
+
+         <button class="btn2">See Project </button> </div>
+        
+                     
+        </section>
+           
+
+
+
+       <section class="body3">
+        <h1 class="s1h11">About me</h1>
+
+        <h3>Hello I’m a software developer! I can help you build a product 
+            , feature or website Look through some of my work and experience! If you like what you see and have
+             a project you need coded, don’t hestiate to contact me.</h3>
+    
+
+          <button class="btn11">Get My Resume </button>  
+              
+             <hr class="hr2">
+             <div class="description">
+                 
+                <div class="des1">
+                   <img src="image/lang.png" alt="">  
```

Comment:
- [ Optional ] You should use a `<h3>...<h6>` tags instead of an image for semantic reasons since these are sub-headings.